easybuild/easyconfigs/d/Dakota/Dakota-6.15.0-foss-2020b.eb

```diff
@@ -0,0 +1,61 @@
+# /!\ IMPORTANT: you need at least 8 cores to complete successfully the tests
+
+easyblock = 'CMakeMake'
+
+name = 'Dakota'
+version = '6.15.0'
+
+homepage = "https://dakota.sandia.gov/"
+description = """
+ The Dakota project delivers both state-of-the-art research and robust, usable
+ software for optimization and UQ. Broadly, the Dakota software's advanced
+ parametric analyses enable design exploration, model calibration, risk
+ analysis, and quantification of margins and uncertainty with computational
+ models.
+"""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+# Disable architecture optimization, otherwise the basic tests fail
+toolchainopts = {'pic': True, 'usempi': True, 'optarch': False}
+
+source_urls = ['https://dakota.sandia.gov/sites/default/files/distributions/public/']
+sources = ["dakota-%(version_major_minor)s.0-public-src-cli.tar.gz"]
+checksums = ['b6d971b7d6fe913aa64fef29cf8fcf55d0e5339f9f55fb164e3cc38716b48f61']
+
+dependencies = [
+    ('Boost',    '1.74.0'),
+    ('OpenBLAS', '0.3.12'),
```

Comment:
OpenBLAS 0.3.12 is already part of foss-2020b. Correct me if I'm wrong, I believe it should be removed.

Reply:
I don't know but as it is quite a sensitive dependency for Dakota (foss vs intel) I would expect it better to kept explicitly.

Reply:
I tried to install it using foss2021a with OpenBLAS as an explicit dependency and the check for banned linked shared libraries failed.

```
Banned patterns found in linked libraries output for /mnt/tier2/apps/USE/easybuild/staging/2021.1/software/Dakota/6.15.0-foss-2021a/bin/dakota_library_split: '(/|\s)libopenblas\.so'
```

I'm just wondering how you didn't face such an issue?

@jfgrimm @branfosj Can you give me some insights on this?

Reply:
Of course, I can give it myself... the reason is that OpenBLAS was a dependency in foss2020b and the check was not necessary.

In foss2021a, it was replaced by FlexiBLAS and the check is introduced.